### DISCUSSION TITLE
Can't verify my academic GitHub account

### DISCUSSION THREAD

**Discussion by karlthere** [community/community#121143] [GitHub Education]
### Hi everyone,

I want to verify my academic GitHub account, but I can't use the back- camera and there is no file upload option. What do I have to do?

**Comment by samus-aran** [comment]
Hi @karlthere 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I can only use my Front Camera to take a photo.](https://github.com/orgs/community/discussions/111352#use-front-camera)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
